packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: url_launcher_platform_interface
+description: A common platform interface for the url_launcher plugin.
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
+version: 0.1.0
```

Comment:
For that same reason, I'm thinking to start this package at 1.0.0

Reply:
Done.